wp-content/themes/core/blocks/core/querynoresults/style.pcss

```diff
@@ -0,0 +1,14 @@
+/**
+ * Styles specific to this block
+ */
+
+.wp-block-query-no-results {
+
+	/**
+	* Hack to get around query block displaying no results blocks
+	* on final page of results
```

Comment:
IIRC, this is a known bug in Gutenberg, right? It would be nice to reference the Issue or PR where they are addressing this so we can check in future versions if this is still needed.

Reply:
Apparently this was merged recently but I'm not seeing it fixed in 6.2.1 - maybe next release? https://github.com/WordPress/gutenberg/pull/53772